src/backend/config/routers.py

```diff
@@ -0,0 +1,88 @@
+from enum import StrEnum
+
+from fastapi import Depends
+
+from backend.database_models import get_session
+from backend.services.auth.request_validators import validate_authorization
+from backend.services.request_validators import (
+    validate_chat_request,
+    validate_user_header,
+)
+
+
+class RouterName(StrEnum):
+    AUTH = "auth"
+    CHAT = "chat"
+    CONVERSATION = "conversation"
+    DEPLOYMENT = "deployment"
+    EXPERIMENTAL_FEATURES = "experimental_features"
+    TOOL = "tool"
+    USER = "user"
+
+
+# Router dependency mappings
+ROUTER_DEPENDENCIES = {
```

Comment:
Can we add some documentation about this? Or maybe tests? Just something to remind people to add dependencies whenever adding a new router.

Reply:
Sure, adding some more documentation